src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator, Drawable {

```

Comment:
Если класс Figure имплементит интерфейсы AreaCalculator, Drawable то все его наследники имплементят их автоматически
```suggestion
public class Circle extends Figure {

```